proteuslib/unit_models/reverse_osmosis_1D.py

```diff
@@ -0,0 +1,800 @@
+###############################################################################
+# ProteusLib Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/nawi-hub/proteuslib/"
+#
+###############################################################################
+
+
+from enum import Enum
```

Comment:
You do not appear to be using this import.

Reply:
I was planning to use this when I add in options for concentration polarization effects and pressure drop assumptions. I could delete now but would be bringing it right back in a subsequent PR (or perhaps just build on this one?)

Reply:
It is generally good practice to only include imports you use. You add the import back once you add the new options.